recipe/scripts/activate.d/cudatoolkit-dev-activate.sh

```diff
@@ -1,102 +1,103 @@
 #!/usr/bin/env bash
+set -o pipefail
 
 create_symlink_linux() { 
 
-        for f in $CONDA_PREFIX/pkgs/cudatoolkit-dev/bin/*;
+        for f in $CONDA_PREFIX/pkgs/cuda-toolkit/bin/*;
```

Comment:
@andersy005 , there is a possibility that downloading cuda installer or when running the installer fails. When such a failure will occur, `$CONDA_PREFIX/pkgs/cuda-toolkit/bin` will not exist and by default, bash will interpret `$CONDA_PREFIX/pkgs/cuda-toolkit/bin/*` as a filename (it does not expand the wildcard). As a result, the `link` value will be `*`, following that the `ln -sf ..` command wil actually read `ln -sf $CONDA_PREFIX/pkgs/cuda-toolkit/bin/*  $CONDA_PREFIX/bin/*` , that will fail with a message `ln: target ... is not a directory` (as in issue #13 , I have seen this also when conda packaging omniscidb-cuda).

Long story short, I suggest using `shopt -s nullglob` before the `for` loop and `shopt -u nullglob` after `done`. As a result, the error messages from any failure of running cuda installer should become clearer.
This applies to all `for` loops in this file.

Reply:
@pearu, thank you for chiming in! I've resolved this. Let me know if there's anything else that needs to be addressed